modules/ROOT/pages/clustering/glossary.adoc

```diff
@@ -0,0 +1,123 @@
+[[cluster-glossary]]
+= Clustering glossary
+:description: This section contains a glossary of terms used in the clustering documentation.
+
+
+[cols="<,<",frame="topbot",options="header"]
+|===
+| Term
+| Description
+
+[[glossary-async-replication]]
+| Asynchronous replication
+| Enables efficient scale-out of secondary database copies but offers no guarantees under fault conditions.
+The data present in the secondary copy is not guaranteed to be up-to-date with a majority of the database's primary copies.
+
+[[glossary-availabiility]]
+| Availability
+| The ability to access data in a database.
+A database can be available for read-write, read-only, or altogether unavailable.
+A clustered database is fault-tolerant, i.e. it can maintain both read and write availability if some primaries fail (see <<glossary-fault-tolerance,Fault tolerance>> for more information).
+If the number of failed primaries exceeds the fault tolerance limit, the database becomes read-only.
+Should all copies fail, the database becomes unavailable.
+
+[[glossary-bookmark]]
+| Bookmark
+| A marker the client can request from the cluster to ensure that it is able to read its own writes so that the application's state is consistent and only databases that have a copy of the bookmark are permitted to respond.
+
+[[glossary-causal-consistency]]
+| Causal consistency
+| When a client (driver) creates a session and executes a query, the responding server issues the client a bookmark.
+This reflects the state of the database copy on that server at the time the query was executed.
+The bookmark is passed along and updated by all subsequent queries in the session, regardless of which server executes what query.
+A bookmark can only be updated monotonically increasing.
+If a server is behind the state in the bookmark, it waits until it has caught up, or time out the query.
+Thus, clients executing queries within a session are guaranteed to read their own writes, and only see successively later states of the database.
+This is sometimes also referred to as <<glossary-session-consistency,session consistency>>.
+
+[[glossary-cluster]]
+| Causal cluster
+| A collection of servers running Neo4j that are configured to communicate with each other.
+The servers can be either _Primary_ or _Secondary_, where the Primary servers allow read and write operations and the Secondary servers allow only read operations.
```

Comment:
Generally the question is if we would not want to use the terms "core" and "read-replica". I think in case of a 4.4 they are completely fine, probably even better as the rest of the docs, would mention those, and not primary and secondary.

Same for below for _Primary server_ and _Secondary server_